kge/util/dump.py

```diff
@@ -113,83 +113,167 @@ def _add_dump_trace_parser(subparsers_dump):
-            "backwards, starting with a specified job_id."
+            "Dump trace to csv (default) and/or stdout. The tracefile will be processed"
+            " backwards starting from the last entry. Further options allow to start"
+            " processing from a particular checkpoint or job_id or epoch_number."
         ),
     )
-
     parser_dump_trace.add_argument(
         "source",
         help="A path to either a checkpoint or a job folder.",
         nargs="?",
         default=".",
     )
-
+    parser_dump_trace.add_argument(
+        "--train",
+        action="store_const",
+        const=True,
+        default=False,
+        help=(
+            "Include entries from training jobs (Note: if none of --train --valid"
+            " --test is set, this will activate all of them jointly)."
+        ),
+    )
+    parser_dump_trace.add_argument(
+        "--valid",
+        action="store_const",
+        const=True,
+        default=False,
+        help=(
+            "Include entries from validation during training and evaluation on"
+            " the validation data split (Note: if none of --train --valid --test is"
+            " set, this will activate all of them jointly)."
+        ),
+    )
+    parser_dump_trace.add_argument(
+        "--test",
+        action="store_const",
+        const=True,
+        default=False,
+        help=(
+            "Include entries from evaluation on the test data split (Note:"
+            " if none of --train --valid --test is set, this will activate"
+            " all of them jointly)."
+        ),
+    )
+    parser_dump_trace.add_argument(
+        "--search",
+        action="store_const",
+        const=True,
+        default=False,
+        help="Dump the tracefile from a search job. If a tracefile from search jobs is"
+        " used, the options --train, --valid and --test are not applicable.",
+    )
+    parser_dump_trace.add_argument(
+        "--keysfile",
+        default=False,
+        help=(
+            "A path to a file which contains lines in the format"
+            " 'new_key_name'='key_name'. For every line in the keys file, the command"
+            " will look for the value of 'key_name' in the trace entries and config and"
+            " add a respective column in the csv file with name 'new_key_name'."
+            " Additionally, for 'key_name' the special keys '$folder', '$machine'"
+            " '$checkpoint' and '$base_model' can be used. "
+        ),
+    )
+    parser_dump_trace.add_argument(
+        "--keys",
+        "-k",
+        nargs="*",
+        type=str,
+        help=(
+            "A list of 'key' entries (separated by space). For every entry,"
+            " the corresponding values of 'key' will be searched in the config and"
+            " trace entries and in the csv file a column with name 'key' will be added."
+            " The same special keys as for --keysfile can be used but must be"
+            " surrounded by quotations ''. "
```

Comment:
This is unlcear. You should say that each KEY is form "KEY_NAME" or "NEW_KEY_NAME=KEY_NAME" and then just refer to the description above